docs/index.md

```diff
@@ -459,6 +459,47 @@ async def app(scope, receive, send):
     })
 ```
 
+### HTTP Trailers
```

Comment:
Do you think it makes sense to be its own section on a new file called "extensions.md"?